app/page.tsx

```diff
@@ -1,14 +1,15 @@
-/* eslint-disable import/no-default-export */
-import React from 'react';
-import { SingleDayLineChart } from '../components/dashboard/charts/SingleDayLineChart';
+'use client';
+
+import { useState } from 'react';
```

Comment:
We should keep React in scope for Typescript
```suggestion
import React, { useState } from 'react';
```